bears/typescript/TSLintBear.py

```diff
@@ -0,0 +1,48 @@
+import json
+
+from coalib.bearlib.abstractions.Lint import Lint
+from coalib.bears.LocalBear import LocalBear
+from coalib.misc.Shell import escape_path_argument
+from coalib.results.Result import Result
+
+
+class TSLintBear(LocalBear, Lint):
+    executable = 'tslint'
+
+    def process_output(self, output, filename, file):
+        try:
+            output = json.loads("".join(output))
+        except ValueError:
+            # This implies that the result is empty and there
```

Comment:
`json.loads("".join(output))` throws a `ValueError`. I can't proceed without catching the exception.